shopify/resources/tender_transaction.py

```diff
@@ -0,0 +1,4 @@
+from ..base import ShopifyResource
+
+class TenderTransaction(ShopifyResource):
+    _prefix_source = "/admin/"
```

Comment:
This line is superfluous since there are no prefix options for this resource

Reply:
👍 I've removed this